cbor.js

```diff
@@ -111,14 +195,18 @@ function encode(value) {
 
     switch (typeof value) {
       case "number":
+        if (isNaN(value) || !isFinite(value)) {
+          return writeFloat16(value);
+        }
```

Comment:
nans and infinities are encoded as half-precision floats under the assumption that the mantissa is not necessary to be preserved, this could be converted to an option or removed